### DISCUSSION TITLE
How to track Digital Signature (DSC) Expiry date

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#613] [QA] [answered]
Suggested by `Prabhu R`

In my office I have about 80 DSC. I want to track the expiry date so that my admin staff can get a new DSC for the Client before it expires.

**Comment by Divwork92** [comment, marked as answer]
You can easily track the expiry dates of DSC or any certificate/document which has got an expiry date.

There are two ways of tracking- 
## 1st (Recommended) From daily dashboard
1. Go to daily dashboard
2. under folder expiry, there are two cards namely - expired and soon on right side
3. click on 'expired' card 

Expired shows list of folder which are expired till date.
Soon shows list of folder which are going to be expired within the next 15 days.

## 2nd From folder list
1. Go to folder list
2. Click on DSC filter or even you can set the expiry date range from-to
3. Click on cloud icon (on extreme right side) to re-load the data

You can see the expired ones in red color
